core/geometry/src/serialization/BGFBAccessors.ts

```diff
@@ -1,6081 +1,6133 @@
-      builder.addFieldInt8(2, +capped, +false);
-    }
-
-    /**
-     * @param flatbuffers.Builder builder
-     * @returns flatbuffers.Offset
-     */
-    static endDgnExtrusion(builder: flatbuffers.Builder): flatbuffers.Offset {
-      const offset = builder.endObject();
-      return offset;
-    }
-
-    static createDgnExtrusion(builder: flatbuffers.Builder, baseCurveOffset: flatbuffers.Offset, extrusionVectorOffset: flatbuffers.Offset, capped: boolean): flatbuffers.Offset {
-      DgnExtrusion.startDgnExtrusion(builder);
-      DgnExtrusion.addBaseCurve(builder, baseCurveOffset);
-      DgnExtrusion.addExtrusionVector(builder, extrusionVectorOffset);
-      DgnExtrusion.addCapped(builder, capped);
-      return DgnExtrusion.endDgnExtrusion(builder);
-    }
-  }
-  /**
-   * @constructor
-   */
-  export class DgnRotationalSweep {
-    bb: flatbuffers.ByteBuffer | null = null;
-
-    bb_pos: number = 0;
-    /**
-     * @param number i
-     * @param flatbuffers.ByteBuffer bb
-     * @returns DgnRotationalSweep
-     */
-    __init(i: number, bb: flatbuffers.ByteBuffer): DgnRotationalSweep {
-      this.bb_pos = i;
-      this.bb = bb;
-      return this;
-    }
-
-    /**
-     * @param flatbuffers.ByteBuffer bb
-     * @param DgnRotationalSweep= obj
-     * @returns DgnRotationalSweep
-     */
-    static getRootAsDgnRotationalSweep(bb: flatbuffers.ByteBuffer, obj?: DgnRotationalSweep): DgnRotationalSweep {
-      return (obj || new DgnRotationalSweep()).__init(bb.readInt32(bb.position()) + bb.position(), bb);
-    }
-
-    /**
-     * @param flatbuffers.ByteBuffer bb
-     * @param DgnRotationalSweep= obj
-     * @returns DgnRotationalSweep
-     */
-    static getSizePrefixedRootAsDgnRotationalSweep(bb: flatbuffers.ByteBuffer, obj?: DgnRotationalSweep): DgnRotationalSweep {
-      bb.setPosition(bb.position() + flatbuffers.SIZE_PREFIX_LENGTH);
-      return (obj || new DgnRotationalSweep()).__init(bb.readInt32(bb.position()) + bb.position(), bb);
-    }
-
-    /**
-     * @param CurveVector= obj
-     * @returns CurveVector|null
-     */
-    baseCurve(obj?: CurveVector): CurveVector | null {
-      const offset = this.bb!.__offset(this.bb_pos, 4);
-      return offset ? (obj || new CurveVector()).__init(this.bb!.__indirect(this.bb_pos + offset), this.bb!) : null;
-    }
-
-    /**
-     * @param DRay3d= obj
-     * @returns DRay3d|null
-     */
-    axis(obj?: DRay3d): DRay3d | null {
-      const offset = this.bb!.__offset(this.bb_pos, 6);
-      return offset ? (obj || new DRay3d()).__init(this.bb_pos + offset, this.bb!) : null;
-    }
-
-    /**
-     * @returns number
-     */
-    sweepRadians(): number {
-      const offset = this.bb!.__offset(this.bb_pos, 8);
-      return offset ? this.bb!.readFloat64(this.bb_pos + offset) : 0.0;
-    }
-
-    /**
-     * @returns number
-     */
-    numVRules(): number {
-      const offset = this.bb!.__offset(this.bb_pos, 10);
-      return
```

Comment:
Oops, these are TBGR values now (originally were color table index).

Reply:
This file is generated by FB compiler in imodel-native using the schema file allcg.fbs over there. I saw that the Polyface color array was incorrectly typed as int[] when it should have been uint[]. So I made this change to allcg.fbs in the native PR. I thought I convinced myself that this was an inconsequential change wrt the FB offsets. Do you see otherwise?

Reply:
Yes, but I'll add an explicit check to verify they round-trip.